ChessboardSquare[64] chessSquare
